libqsopt/CMakeLists.txt

```diff
@@ -20,17 +23,29 @@ if(CMAKE_DPKG)
   endif()
 endif()
 
-
 if(SUPPORTED MATCHES "true")
+  file(WRITE ${CATKIN_DEVEL_PREFIX}/${CATKIN_PACKAGE_INCLUDE_DESTINATION}/qsopt.h "")  # create dummy file for correct configuration of libconcorde_tsp_solver
+
   catkin_package(
+    INCLUDE_DIRS
+      common/lib/${MYARCH}
     CFG_EXTRAS libqsopt-extras.cmake # catkin_package does not support artifacts in devel space
   )
 
-  install(FILES common/lib/${MYARCH}/qsopt.a
-    DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
+  # Concorde expects qsopt.a to be in the same directory as qsopt.h --> static library, when using in another package one has to provide an absolut path to it
+  add_custom_target(
+    build_qsopt ALL
+      # copy headers to devel space (catkin does not like headers in source space)
+      COMMAND ${CMAKE_COMMAND} -E copy_if_different ${PROJECT_SOURCE_DIR}/common/lib/${MYARCH}/qsopt.h ${CATKIN_DEVEL_PREFIX}/${CATKIN_PACKAGE_INCLUDE_DESTINATION}
+      # copy static lib to include destination as well because concorde needs both files in the same folder
+      COMMAND ${CMAKE_COMMAND} -E copy_if_different ${PROJECT_SOURCE_DIR}/common/lib/${MYARCH}/qsopt.a ${CATKIN_DEVEL_PREFIX}/${CATKIN_PACKAGE_INCLUDE_DESTINATION}
```

Comment:
@ipa-rmb 
try `CATKIN_PACKAGE_LIB_DESTINATION` for the destination of the static library...and adjust install tags, too

Reply:
as written in libconcorde_tsp_solver/CMakeLists.txt in line 11:
# you need to have a qsopt.a and a qsopt.h in the source folder and you have to use an absolute path to link it with Concorde

I tried to get them in include and lib separately, but it does not work with libconcorde_tsp_solver and I cannot change the binary.